src/EnergyPlus/WaterThermalTanks.cc

```diff
@@ -8820,6 +8790,10 @@ namespace WaterThermalTanks {
             DeadBandTempDiff = max(0.0, (SetPointTemp - CutInTemp));
         }
 
+        Real64 Acc; // Accuracy of result from RegulaFalsi
+        if (WaterHeaterDesuperheater(DesuperheaterNum).TankTypeNum == StratifiedWaterHeater) Acc = 0.001;
+        else Acc = 0.00001;
```

Comment:
Since you'll be making other changes to add the new test file, this if/else block should have braces { }